fga-eps-mds/2020.1-stay-safe-front-end
# Standardizing code

## Issue 
Resolves #31 


## Description

Standardizes all necessary repository code.

## Checklist:



## How to validate?

Testing to see if all screens are working as expected. Check if the code follows the ESLint style and the same pattern of files, functions, components, etc.
